libres/lib/enkf/enkf_main.cpp

```diff
@@ -706,20 +702,20 @@ void init_active_runs(const res_config_type *res_config,
 }
 
 /**
- * @brief clears runpath_list and adds all active runs to it.
+ * @return list of the runpaths in the runcontext.
  */
-void reset_run_path(runpath_list_type *runpath_list,
-                    const ert_run_context_type *run_context) {
-    runpath_list_clear(runpath_list);
+std::vector<Runpath>
+run_context_get_runpath_list(const ert_run_context_type *run_context) {
```

Comment:
`run_context_get_runpaths` instead? We're not returning a list in the C++ sense (ie. linked list) nor a Python `list`.

Reply:
Done